### DISCUSSION TITLE
Material Theme?

### DISCUSSION THREAD

**Discussion by paarthjuneja** [monkeytypegame/monkeytype#1653] [Ideas] [answered]
I think a material theme would be great considering the fact that it is a such a popular theme.
Thanks

**Comment by Miodec** [comment, marked as answer]
Send an example of what you have in mind

**Reply by paarthjuneja** [reply]
I'm thinking of having it similar to the [Visual Studio Code material theme](https://marketplace.visualstudio.com/items?itemName=Equinusocio.vsc-material-theme). I'm attaching a screenshot below.
![Screenshot_2021-07-24-29_1344x718](https://user-images.githubusercontent.com/77548507/126869761-ead63f26-9241-49ff-bad6-e507509deb52.png)
The text and icons could be similar to the light blue shades in the screenshot and the bg color could be the same.
Anything else?
Thanks

**Reply by Miodec** [reply]
Alright, added.
